keda/templates/manager/clusterrole.yaml

```diff
@@ -26,77 +30,59 @@ rules:
   - events
   verbs:
   - '*'
+{{- if not .Values.permissions.operator.restrict.secret }}
 - apiGroups:
   - ""
-  resources:
-  - external
-  - pods
-    {{- if eq .Values.permissions.operator.restrict.secret false }}
+  resources:    
   - secrets
-    {{- end }}
-  - services
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - limitranges
   verbs:
   - list
   - watch
+  {{- with .Values.permissions.operator.restrict.namesAllowList }}
 - apiGroups:
   - ""
   resources:
-  - serviceaccounts
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - '*'
-  resources:
-  - '*'
+  - secrets
   verbs:
   - get
+  resourceNames: {{ toYaml . | nindent 4 }}
+  {{- end }}
+{{- end }}
+
+{{- if .Values.rbac.enabledCustomScaledRefKinds }}
+{{- if not .Values.rbac.scaledRefKinds }}
 - apiGroups:
   - '*'
   resources:
   - '*/scale'
   verbs:
-  - get
-  - list
-  - patch
-  - update
-  - watch
-  {{- if and .Values.certificates.autoGenerated ( not .Values.certificates.certManager.enabled ) }}
```

Comment:
`admissionregistration.k8s.io` and `apiregistration.k8s.io` permissions are required in the operator when certificates are managed internally. I've seen that you moved this permission from operator to webhooks but the operator is the correct place for it

Reply:
for operator, it's still there in the `minimal-rbac.yaml` (because it's a cluster-wide resource). I wasn't sure if this is needed by webhook itself or not so I left it there just for sure, but I will remove it. I've double checked and operator needs this for cert rotation and when installing the webhook the `ValidatingWebhookConfiguration` is created as a normal resource by helm and applied by helm (so probably someone with superpowers rbac :D)